docs/converters/vision_converter.md

```diff
@@ -56,9 +56,12 @@ An examplery usage can be found in the example notebook as well as in the test c
 So far we have the following issue known to us related to Vision exported spread sheets. We provide a solution from user perspective to the best of our knowledge.
 
 ### Duplicated `P` columns
-Vision can export sheets with duplicated `P` columns, one of which being unitless additional information. This field is of no actual purpose within PGM calculation. 
+Vision can export sheets with duplicated `P` columns, one of which being unitless additional information. This field is of no actual purpose within PGM calculation. In case of duplicate column names, PGM-IO renames them to eg. `P`, `P_2` and so on. However the change in mapping to the duplicate name should be done manually by the user in a custom mapping file.
```

Comment:
```suggestion
Vision can export sheets with duplicated `P` columns, one of which being unitless additional information. This field is of no actual purpose within PGM calculation. In case of duplicate column names, PGM-IO renames them e.g., from `P` to `P_2` and so on. However, the change in mapping to the duplicate name should be done manually by the user in a custom mapping file.
```

Reply:
How about this?

```suggestion
Vision can export sheets with duplicated `P` columns, one of which being unitless additional information. This field is of no actual purpose within PGM calculation. In case of duplicate column names, PGM-IO renames them. The renaming is done in following way for example in case of a single duplicate:  `P` and `P_2`. However, the change in mapping to the duplicate name should be done manually by the user in a custom mapping file.
```